### DISCUSSION TITLE
Open a modal after clicking an issue, not a new tab

### DISCUSSION THREAD

**Discussion by markustoivonen** [community/community#10071] [Projects and Issues]
When clicking issues on the project page, I wish the application would open a modal that shows the ticket (similar to Zenhub) instead of opening the ticket on a new tab. Would make using the board view much smoother.

**Comment by jaredsb** [comment]
Totally agree. Let us know what you think of the [Draft Issue modal expereince](https://github.blog/changelog/2021-12-16-the-new-github-issues-12-16-update/), and expect to see more work in this area coming soon!

**Reply by adammesa** [reply]
Definitely love the modal experience. It severely hampers the reviewing experience to be constantly taken to a new page rather than through a modal. I was shocked when I made something not a draft and suddenly the modal stopped working.

**Comment by markustoivonen** [comment]
@jaredsd not sure if you want feedback on this thread or if I should open up another regarding the draft issue modal but here goes:

- Would love to see support for templates (although as templates change name of ticket as well, wouldn't fit into current flow where you name the ticket first and then you would change it with a template) 
- Would love to see the possibility to add labels in the draft view

Overall I wish there was a button that would open up a modal editor instead of first writing the ticket name and then opening the modal. Although it is a nice touch that when you write the draft ticket, you add it straight to the section you want to add it to.

**Reply by jaredsb** [reply]
Hi @markustoivonen I think you were directing these comments at me and not at `jaredSD` so I'll follow up.

> Would love to see support for templates (although as templates change name of ticket as well, wouldn't fit into current flow where you name the ticket first and then you would change it with a template)

This is a topic we are looking at and we should be picking up later this year. 

> Would love to see the possibility to add labels in the draft view

Labels are repo scoped and since drafts aren't in a repo we don't have an easy way to connect the two. This is something we are looking at, but I wouldn't expect to add labels to drafts in the near future.

**Reply by markustoivonen** [reply]
Oh whoops, glad that you still found the comment though! @jaredsb 

Thank you for the answers!

One more comment (sorry, I know this is not the optimal way to receive feedback from your POV) regarding Projects and Issues. When creating an issue and adding a project to it, I wish I could set the lane in the editor itself and not **after** I've created the ticket.
